xrmocap/human_perception/keypoints_estimation/mediapipe_estimator.py

```diff
@@ -0,0 +1,246 @@
+import cv2
+import logging
+import mediapipe as mp
+import numpy as np
+from tqdm import tqdm
+from typing import Tuple, Union
+from xrprimer.utils.ffmpeg_utils import video_to_array
+from xrprimer.utils.log_utils import get_logger
+
+from xrmocap.transform.convention.keypoints_convention import get_keypoint_num
+from .mmpose_top_down_estimator import MMposeTopDownEstimator
+
+
+class MediapipeEstimator(MMposeTopDownEstimator):
```

Comment:
It has few common functions as `MMposeTopDownEstimator`, we donot have to sub-class it.

Reply:
The only common function is `get_keypoints_from_result`. Should we inherit it from `MMposeTopDownEstimator `?